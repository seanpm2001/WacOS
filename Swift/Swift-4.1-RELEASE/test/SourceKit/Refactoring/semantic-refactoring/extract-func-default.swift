func foo() -> Int {
  var a = 3
  a = a + 1
  return 1
}

// RUN: rm -rf %t.result && mkdir -p %t.result
// RUN: %sourcekitd-test -req=extract-func -pos=2:1 -end-pos 4:11 %s -- %s > %t.result/extract-func-default.swift.expected
// RUN: diff -u %S/extract-func-default.swift.expected %t.result/extract-func-default.swift.expected

// FIXME: Fails on linux with assertion: "!GlibcModuleMapPath.empty()"" failed
// REQUIRES-ANY: OS=macosx
