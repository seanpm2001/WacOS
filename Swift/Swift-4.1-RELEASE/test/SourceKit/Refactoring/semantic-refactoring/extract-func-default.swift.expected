source.edit.kind.active:
  1:1-1:1 "fileprivate func extractedFunc() -> Int {
var a = 3
  a = a + 1
  return 1
}

"
  <note>source.refactoring.range.kind.basename 1:18-1:31 "extractedFunc"</note>
source.edit.kind.active:
  2:3-4:11 "return extractedFunc()"
  <note>source.refactoring.range.kind.basename 1:8-1:21 "extractedFunc"</note>
